flutter_common/lib/data/environment/environment_info.dart

```diff
@@ -31,3 +33,51 @@ class EnvironmentInfo {
         .key;
   }
 }
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class EnvSocInfo {
+  final EnvCpuinfo cpuinfo;
```

Comment:
I used cpuinfo as a single word here because it's the name of the library.
It's not "CPU Info", it's "data obtained from cpuinto lib".

Though, I don't have a strong preference here. If you think lib name shouldn't be used as a word here, I don't mind changing it.

Reply:
We are writing Dart code here, so I think matching style `cpuInfo` would be better. If you want to say "data obtained from cpuinfo lib" then a comment is better.

Reply:
Would it be enough to rename this field to `libCpuinfo`?

Reply:
Either `cpuInfo` or `cpuinfo` I understand them both as  "info about cpu". I assume others may also understand so. So in this case it should have a different name to avoid confusion.

Reply:
> Would it be enough to rename this field to `libCpuinfo`?

I propose `libCpuinfo`.
It would be clear that this is not some general info but an info from a specific source.

We could use `libCpuInfo` if you insist on using proper English words.
I would prefer to keep the `cpuinfo` reference in the name but I would be OK with changing it.